config/cmake/updateGitSubmodules.cmake

```diff
@@ -47,4 +47,30 @@ macro(submod_update target)
    else()
      message(WARNING "SUBMODULE update has been disabled")
    endif()
-endmacro()
\ No newline at end of file
+endmacro()
+
+function(check_submodule_status)
+    if (ENABLE_SUBMODULE_UPDATE)
```

Comment:
Should this variable be made HELICS specific list ${PROJECT_NAME}_ENABLE_SUBMODULE_UPDATE ?  @nightlark @eberleim